inmysparetime.html

```diff
@@ -97,6 +123,35 @@ <h3 align='center'> Coding! </h1>
     <a href="https://twitter.com/beverleynewing" target="_blank">
         <img src="https://simplesharebuttons.com/images/somacro/twitter.png" alt="Twitter" />
     </a>
+
+<script>
```

Comment:
I'd forgotten about that! I will do that page separately as a revision exercise on the modal stuff here.